worlds/sc2/Items.py

```diff
@@ -68,175 +213,807 @@ def get_full_item_list():
+                                     """),
+
+    ItemNames.WAR_PIGS: ItemData(500 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 0, SC2Race.TERRAN,
+                                 classification=ItemClassification.progression,
+                                 description="Mercenary Marines"),
+    ItemNames.DEVIL_DOGS: ItemData(501 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 1, SC2Race.TERRAN,
+                                   classification=ItemClassification.filler,
+                                   description="Mercenary Firebats"),
+    ItemNames.HAMMER_SECURITIES: ItemData(502 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 2, SC2Race.TERRAN,
+                                          description="Mercenary Marauders"),
+    ItemNames.SPARTAN_COMPANY: ItemData(503 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 3, SC2Race.TERRAN,
+                                        classification=ItemClassification.progression,
+                                        description="Mercenary Goliaths"),
+    ItemNames.SIEGE_BREAKERS: ItemData(504 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 4, SC2Race.TERRAN,
+                                       description="Mercenary Siege Tanks"),
+    ItemNames.HELS_ANGEL: ItemData(505 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 5, SC2Race.TERRAN,
+                                   classification=ItemClassification.progression,
+                                   description="Mercenary Vikings"),
+    ItemNames.DUSK_WINGS: ItemData(506 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 6, SC2Race.TERRAN,
+                                   description="Mercenary Banshees"),
+    ItemNames.JACKSONS_REVENGE: ItemData(507 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 7, SC2Race.TERRAN,
+                                         description="Mercenary Battlecruiser"),
+
+    ItemNames.ULTRA_CAPACITORS: ItemData(600 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 0, SC2Race.TERRAN,
+                                         description="""
+                                         Increases attack speed of all Terran units with each weapon upgrade.
+                                         """),
+    ItemNames.VANADIUM_PLATING: ItemData(601 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 1, SC2Race.TERRAN,
+                                         description="""
+                                         Increases life of all Teran units with each armor upgrade.
+                                         """),
+    ItemNames.ORBITAL_DEPOTS: ItemData(602 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 2, SC2Race.TERRAN,
+                                       description="""
+                                       Supply depots are built instantly.
+                                       """),
+    ItemNames.MICRO_FILTERING: ItemData(603 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 3, SC2Race.TERRAN,
+                                        description="""
+                                        Increases Vespene Gas harvesting speed.
```

Comment:
Based on GameStrings.txt (it starts with "this refinery", replaced with "Refineries")
> Refineries produce Vespene gas 25% faster.